lib/ecto/adapters/clickhouse/structure.ex

```diff
@@ -82,30 +81,25 @@ defmodule Ecto.Adapters.ClickHouse.Structure do
     end
   end
 
-  defp run_with_cmd(cmd, opts, opt_args, cmd_opts \\ []) do
+  defp clickhouse_client_cmd do
+    case :os.type() do
+      {_, :darwin} -> {"clickhouse", _args = ["client"]}
```

Comment:
For some reason on mac it's `clickhouse` and `server` / `client` are args.

Reply:
I think it depends on the version or maybe if installed via homebrew? 

I have `clickhouse-client` & `clickhouse client`

Reply:
I wonder if we should use a package manager and install only `clickhouse-client`.

Reply:
Maybe we have to try on darwin both options? There will be users having clickhouse and others having clickhouse-client

Reply:
It seems like it applies to more than just mac. I'll change that part to try clienthouse-client first and fallback to clickhouse if it's not found.

Reply:
Done in https://github.com/Zarathustra2/chto/pull/1/commits/f8f7a90418d9359b729fcfec3ec840a468d5172a

Reply:
awesome, I will merge this then, okay?

Reply:
Okay :)